docs/user-guides/creating-a-simple-ca-based-issuer.md

```diff
@@ -0,0 +1,53 @@
+# Creating a simple CA based issuer
+
+`cert-manager` can be used to obtain certificates using a CA key pair. The first two steps [here][1] from the Kubernetes documentation describe how you can generate a self-signed CA key pair using [openssl][2]. The output of these commands will be two files, `ca.key` and `ca.crt`, the key and certificate for your CA.
```

Comment:
`CA key pair` -> `signing key pair`

Reply:
Can we not link out to external docs for instructions on using openssl? Instead include it here. Too often the URLs change and we have broken links, plus it's only 1 or 2 commands anyway. Easier reading, less jumping about, less confusion 😄

Reply:
Also when updating this to not link out, it'd be good if we mention that you can use `openssl` or `cfssl` or a number of other utilities